docs/source/guides/monitor-iroha-peer.md

```diff
@@ -0,0 +1,37 @@
+# How to Monitor Iroha Peer
+
+When you need to monitor state and work of Iroha peer, use this guide.
+
+## Prerequisites
+
+* Iroha CLI
+
+## Steps
+
+### 1. Run CLI Command to check Peer's Health
+
+```bash
+./iroha_client_cli maintenance health
```

Comment:
I think we should also include a section on how to build and configure iroha_client_cli

Reply:
It's covered under Iroha Client CLI readme

Reply:
Oh, yes, I forgot, then maybe add a link to readme here?